dotnet-sso-example/wwwroot/css/login.css

```diff
@@ -0,0 +1,223 @@
+body {
+  font-family: Inter, sans-serif;
```

Comment:
The CSS on the login page looks a little wonky
![CleanShot 2022-07-11 at 10 55 18](https://user-images.githubusercontent.com/47123183/178317467-0c0e952d-1304-4255-979a-e60f6f38c43b.png)